src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts

```diff
@@ -679,12 +679,27 @@ export class UbsAdminTariffsDeactivatePopUpComponent implements OnInit, OnDestro
   }
 
   public createDeactivateCardDto() {
-    this.deactivateCardObj = {
-      courierId: this.selectedCourier.id,
-      receivingStationsIdList: this.selectedStations.map((it) => it.id),
-      regionId: this.selectedRegions.map((it) => it.id),
-      locationIdList: this.selectedCities.map((it) => it.id)
+    const arr = [];
+    const deactivateCardObj = {
+      cities: this.selectedCities.map((it) => it.id).join('%'),
+      courier: this.selectedCourier?.id,
+      regions: this.selectedRegions.map((it) => it.id).join('%'),
+      stations: this.selectedStations.map((it) => it.id).join('%')
     };
+
+    const requestObj = {
+      cities: `citiesId=${deactivateCardObj.cities}`,
+      courier: `courierId=${deactivateCardObj.courier}`,
+      regions: `regionsId=${deactivateCardObj.regions}`,
+      stations: `stationsId=${deactivateCardObj.stations}`
+    };
+
+    for (const key in deactivateCardObj) {
+      if (deactivateCardObj[key]) {
+        arr.push(requestObj[key]);
+      }
+    }
+    this.query += arr.join('&');
```

Comment:
Move this to service, it is its responsibility to build request.

Reply:
Thanks, I have fixed it